ide/libs.flexmark/nbproject/project.properties

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-javac.source=1.8
+javac.source=11
```

Comment:
I think this can be reverted as only the sigtest is disabled.

Reply:
Reverted.